public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx

```diff
@@ -148,47 +152,60 @@ export class GeneralSettingsEditView
     const { intervals } = model.getRefreshPicker().useState();
     const { hideTimeControls } = model.getDashboardControls().useState();
 
+    const [titleKey, setTitleKey] = useState(uuidv4());
+    const [descriptionKey, setDescriptionKey] = useState(uuidv4());
```

Comment:
@ivanortegaalba this is a bit hacky solution because we don't control the input value. Alternative would be 
changing 
```typescript
<Input
                id="title-input"
                name="title"
                defaultValue={title}
                key={titleKey}
                onBlur={(e: ChangeEvent<HTMLInputElement>) => model.onTitleChange(e.target.value)}
              />
```
to 
```typescript
<Input
                id="title-input"
                name="title"
                value={title}
                onChange={(e: ChangeEvent<HTMLInputElement>) => model.onTitleChange(e.target.value)}
              />
```
`onChange` approach is cleaner but calls state change on every type